### PR TITLE
Update TS definitions

### PR DESCRIPTION
**Pull request by LKay**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
   },
   "ava": {
     "require": [
-      "babel-register"
+      "babel-register",
+      "babel-polyfill"
```
Comment (rt2zz) [inline package.json]: why is babel-polyfill needed?

File: package.json [hunk 2]
```diff
@@ -42,25 +43,25 @@
   "license": "MIT",
   "dependencies": {
     "json-stringify-safe": "^5.0.1",
-    "lodash": "^4.11.1",
-    "lodash-es": "^4.16.0"
+    "lodash": "^4.17.4",
+    "lodash-es": "^4.17.4",
+    "redux": "^3.6.0"
```
Comment (rt2zz) [inline package.json]: redux should be a devDependency.

File: package.json (hunk 1 above)
Comment (LKay) [inline package.json]: Babel core slightly changed and is not injecting `runtimeGenerator` for async functionality by default. It is necessary to either use `babel-polyfill` or `babel-plugin-transform-runtime` to inject proper polyfill. https://babeljs.io/docs/plugins/transform-runtime/

This is required to ava run tests asynchronously.

**Comment by LKay** [comment]
@rt2zz Any chance it (and the update for compress package) will be merged and released soon?

**Comment by rt2zz** [comment]
@LKay done, released 4.4 and 4.2 respectively 🎈

**Comment by marvinhagemeister** [comment]
@LKay awesome! Much appreciated!

**Pull request merged**